vstruct/types.py

```diff
@@ -198,7 +198,7 @@ def __init__(self, size=0):
         vs_bases.v_prim.__init__(self, size=size, valu=b'')
 
     def _prim_emit(self, x):
-        return x
+        return x.ljust(self.vsSize(), b'\x00')
```

Comment:
what is the intended behavior if a user tries to stuff more than `size` bytes into the instance? should it be truncated or allowed? don't have to address this now, but something to think about.